src/client/java/minicraft/core/Renderer.java

```diff
@@ -63,7 +63,10 @@ private Renderer() {
 	static float SCALE = 3;
 
 	public static Screen screen; // Creates the main screen
-	public static SpriteLinker spriteLinker; // The sprite linker for sprites
+	public static final SpriteLinker spriteLinker; // The sprite linker for sprites
+	static {
+		spriteLinker = new SpriteLinker();
+	}
```

Comment:
Okay, this is just plain out stupid. Just make it a static initaliser then, no need to create a static block. Functionally the same as `public static final SpriteLinker spriteLinker = new SpriteLinker();`

Reply:
Right.